lib/minitest/reporters/spec_reporter.rb

```diff
@@ -10,6 +10,16 @@ class SpecReporter < BaseReporter
       include ANSI::Code
       include RelativePosition
 
+      # The constructor takes an `options` hash
+      # @param options [Hash]
+      # @option options print_at_bottom [Boolean] wether to print the errors at the bottom of the
+      #   report or inline as they happen.
+      #
+      def initialize(options = {})
+        super
+        @print_at_bottom = options[:print_at_bottom]
```

Comment:
I'm not convinced about the name for the option here, so suggestions are very welcome :smile: